web/src/features/dialog/InputDialog.tsx

```diff
@@ -104,9 +112,26 @@ const InputDialog: React.FC = () => {
                 {row.type === "input" && (
                   <Box mb={3} key={`input-${index}`} textAlign="left">
                     <Text>{row.label}</Text>
-                    <Input
-                      onChange={(e) => handleChange(e.target.value, index)}
-                    />
+                    <InputGroup>
+                      <Input
+                        onChange={(e) => handleChange(e.target.value, index)}
+                        type={!row.password || showPassword ? 'text' : 'password'}
+                      />
+                      {row.password && (
+                        <InputRightElement
+                          cursor="pointer"
+                          onClick={handleShowPassword}
```

Comment:
Wouldn't this show the password on all input fields that have the password property on them?

Reply:
Yees, fixed in the next commit.